streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -701,10 +701,13 @@ private void runLoop() {
                     enforceRebalance();
                 }
             } catch (final TaskMigratedException ignoreAndRejoinGroup) {
-                log.warn("Detected task {} that got migrated to another thread. " +
-                        "This implies that this thread missed a rebalance and dropped out of the consumer group. " +
-                        "Will try to rejoin the consumer group. Below is the detailed description of the task:\n{}",
-                    ignoreAndRejoinGroup.migratedTask().id(), ignoreAndRejoinGroup.migratedTask().toString(">"));
+                log.warn("Detected task " + ignoreAndRejoinGroup.migratedTask().id() +
+                             " that got migrated to another thread. This implies that this thread missed" +
+                             " a rebalance and dropped out of the consumer group. Will try to rejoin the" +
+                             " consumer group. Below is the detailed description of the task:\n" +
+                             ignoreAndRejoinGroup.migratedTask().toString(">"),
+                         ignoreAndRejoinGroup
+                         );
```

Comment:
I added the exception to the log message, so we can see where the TaskMigratedException is actually caused.